libs/verifiable-credentials/src/main/java/uk/gov/di/ipv/core/library/verifiablecredential/helpers/VcHelper.java

```diff
@@ -225,9 +209,9 @@ private static Integer getAge(String dobValue) {
         }
     }
 
-    public static Vot getVcVot(SignedJWT vc) throws UnrecognisedVotException {
+    public static Vot getVcVot(VerifiableCredential vc) throws UnrecognisedVotException {
```

Comment:
(Not related to your change, but I wonder if we should always use this instead of the version in `UserIdentityService` - we'll need to handle `null` instead of just relying on an exception...)

Reply:
I can make that change if you want? I think the null case is already handled because it's only used here:
```
        try {
            var hmrcMigrationVc = verifiableCredentialService.getVc(userId, HMRC_MIGRATION_CRI);
            if (hmrcMigrationVc == null) {
                return false;
            }

            var existingInheritedIdentityVot = userIdentityService.getVot(hmrcMigrationVc);
            var incomingInheritedIdentityVot =
                    userIdentityService.getVot(incomingInheritedIdentity);

            var indexOfExistingVot =
                    HMRC_PROFILES_BY_STRENGTH.indexOf(existingInheritedIdentityVot);
            var indexOfIncomingVot =
                    HMRC_PROFILES_BY_STRENGTH.indexOf(incomingInheritedIdentityVot);

            if (indexOfExistingVot == -1 || indexOfIncomingVot == -1) {
                throw new IllegalArgumentException(
                        String.format(
                                "At least one of the existing (%s) or incoming (%s) VoTs from hmrc aren't expected",
                                existingInheritedIdentityVot, incomingInheritedIdentityVot));
            }

            return indexOfIncomingVot > indexOfExistingVot;
        } catch (ParseException | IllegalArgumentException e) {
            throw new CredentialParseException(
                    "Encountered a parsing error while attempting to parse credentials", e);
        }
```

Where the nullpointerexception is caught

Reply:
I feel like catching a `NullPointerException` is always a bit of a code smell!